fendermint/vm/interpreter/src/fvm/state/genesis.rs

```diff
@@ -122,6 +129,48 @@ where
             store,
             multi_engine,
             stage: Stage::Tree(state_tree),
+
+            builtin_actor_bundle: vec![],
+            custom_actor_bundle: vec![],
+        };
+
+        Ok(state)
+    }
+
+    pub async fn new_state(
```

Comment:
Since we no longer parse the bundles here, consider renaming to something like `new_empty_state`